src/css/icons.styles.ts

```diff
@@ -0,0 +1,21 @@
+// icon styles, setting general height and width properties to solves scaling and consistency problems
```

Comment:
don't add these here, we have a constant directory, define theme there accordingly

Reply:
I see. Will do so and update the PR.

Reply:
Done and updated the PR @sudhanshutech